File: CadastroPessoaBack/CadastroPessoaBack/Data/Pessoa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CadastroPessoaBack.Data
{
    public class Pessoa
    {
        internal object endereco;

        public virtual int Id { get; set; }
        public virtual string Nome { get; set; }
        public virtual string Cpf { get; set; }
        public virtual string Email { get; set; }
        public virtual DateTime DataNascimento { get; set; }
        public virtual ICollection<Endereco> Enderecos { get; set; }

    }
}

File: CadastroFront/src/app/models/Pessoa.ts
import { Endereco } from "./Endereco";

export class Pessoa {
    [x: string]: any;
    id!: number;
    nome!: string;
    cpf!: string;
    email!: string;
    dataNascimento!: Date;
    enderecos!: Endereco[];
}
